docs/api/browser-window.md

```diff
@@ -70,6 +70,13 @@ Properties `width` and `height` are required.
   Linux.
 * `standard-window` Boolean - Uses the OS X's standard window instead of the
   textured window. Defaults to `true`.
+* `title-bar-style` String, OS X - specifies the style of window title bar. This option is supported on OS X 10.10 Yosemite and newer. There are three possible values:
```

Comment:
From doc [`stylecode`](https://github.com/atom/electron/blob/master/docs/styleguide.md), the line length <= 80.